psutil/_psutil_windows.c

```diff
@@ -601,7 +601,7 @@ psutil_proc_memory_info(PyObject *self, PyObject *args)
         return NULL;
     }
 
-    if (! GetProcessMemoryInfo(hProcess, &cnt, sizeof(cnt)) ) {
+    if (! GetProcessMemoryInfo(hProcess, (PPROCESS_MEMORY_COUNTERS)&cnt, sizeof(cnt)) ) {
```

Comment:
This line len exceeds 80 chars. Something like this should work:

    if (! GetProcessMemoryInfo(hProcess, (PPROCESS_MEMORY_COUNTERS)&cnt, 
                                                 sizeof(cnt)) ) 
    {